music_assistant/controllers/cache.py

```diff
@@ -148,7 +149,7 @@ async def set(
         expiration: int = DEFAULT_CACHE_EXPIRATION,
         provider: str = "default",
         category: int = 0,
-        checksum: str | None = None,
+        checksum: str | int | None = None,
```

Comment:
```suggestion
        checksum: str | None = None,
```

checksum must always be str (the discussion was just about other places in the code where we might be sending this as an int)